Part1-MetaModel-Schemas/openapi.yaml

```diff
@@ -794,177 +778,170 @@ components:
+              $ref: "#/components/schemas/DataTypeDefXsd"
+            value:
+              type: array
+              items:
+                $ref: "#/components/schemas/SubmodelElement_choice"
+              minItems: 1
+            modelType:
+              pattern: "^SubmodelElementList$"
+          required:
+            - typeValueListElement
     SubmodelElement_choice:
       oneOf:
-      - "$ref": "#/components/schemas/RelationshipElement"
-      - "$ref": "#/components/schemas/AnnotatedRelationshipElement"
-      - "$ref": "#/components/schemas/BasicEventElement"
-      - "$ref": "#/components/schemas/Blob"
-      - "$ref": "#/components/schemas/Capability"
-      - "$ref": "#/components/schemas/Entity"
-      - "$ref": "#/components/schemas/File"
-      - "$ref": "#/components/schemas/MultiLanguageProperty"
-      - "$ref": "#/components/schemas/Operation"
-      - "$ref": "#/components/schemas/Property"
-      - "$ref": "#/components/schemas/Range"
-      - "$ref": "#/components/schemas/ReferenceElement"
-      - "$ref": "#/components/schemas/SubmodelElementCollection"
-      - "$ref": "#/components/schemas/SubmodelElementList"
+        - $ref: "#/components/schemas/RelationshipElement"
+        - $ref: "#/components/schemas/AnnotatedRelationshipElement"
+        - $ref: "#/components/schemas/BasicEventElement"
+        - $ref: "#/components/schemas/Blob"
+        - $ref: "#/components/schemas/Capability"
+        - $ref: "#/components/schemas/Entity"
+        - $ref: "#/components/schemas/File"
+        - $ref: "#/components/schemas/MultiLanguageProperty"
+        - $ref: "#/components/schemas/Operation"
+        - $ref: "#/components/schemas/Property"
+        - $ref: "#/components/schemas/Range"
+        - $ref: "#/components/schemas/ReferenceElement"
+        - $ref: "#/components/schemas/SubmodelElementCollection"
+        - $ref: "#/components/schemas/SubmodelElementList"
     ValueList:
       type: object
       properties:
         valueReferencePairs:
           type: array
           items:
-            "$ref": "#/components/schemas/ValueReferencePair"
+            $ref: "#/components/schemas/ValueReferencePair"
           minItems: 1
       required:
-      - valueReferencePairs
+        - valueReferencePairs
     ValueReferencePair:
       type: object
       properties:
         value:
           type: string
           minLength: 1
-          maxLength: 2000
-          pattern: "^([\\x09\\x0a\\x0d\\x20-\\ud7ff\\ue000-\\ufffd]|\\ud800[\\udc00-\\udfff]|[\\ud801-\\udbfe][\\udc00-\\udfff]|\\udbff[\\udc00-\\udfff])*$"
+          maxLength: 2048
+          pattern: ^([\x09\x0a\x0d\x20-\ud7ff\ue000-\ufffd]|\ud800[\udc00-\udfff]|[\ud801-\udbfe][\udc00-\udfff]|\udbff[\udc00-\udfff])*$
         valueId:
-          "$ref": "#/components/schemas/Reference"
+          $ref: "#/components/schemas/Reference"
       required:
-      - value
-      - valueId
\ No newline at end of file
+        - value
+        - valueId
```

Comment:
ValueReferencePait/valueId is optional (0..1)

Reply:
But then it's also wrong in the current JSON Schema (https://github.com/admin-shell-io/aas-specs/blob/3a53dcebfffffac285a8db548c32c8a84d9f46f0/schemas/json/aas.json#L1522), or?